packages/core/src/Stores/client-store.js

```diff
@@ -581,6 +581,11 @@ export default class ClientStore extends BaseStore {
         return this.residence === 'gb';
     }
 
+    @computed
+    get is_belgium() {
```

Comment:
Below the `is_belgium` function you created, there is an already existing function called `country_standpoint`. This function includes the belgium check already, isn't it perhaps better to use this function instead of making a new one? You could do `country_standpoint.is_belgium`

Reply:
Nice. will do.